terraform-plans/templates/github/SECURITY.md.tftpl

```diff
@@ -0,0 +1,24 @@
+# This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation
+# To update the file:
+# - Edit it in the canonical/solutions-engineering-automation repository.
+# - Open a PR with the changes.
+# - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
+
+# Security policy
+
+If the vulnerability affects a dependency, a new version of the component including the updated
+dependency will be released in the respective store, meaning that no new feature will be included:
+the update will be built on top of the previously last released stable version.
+
+If the vulnerability affects our charm/snap code itself, a new version will be built including the
+security fix on top of the current main branch, meaning that the security update will potentially
+include new previously unreleased features.
```

Comment:
Is this a new policy, or existing? If the latter, could you link to our docs on this? If the former, could we discuss it further? :)

Reply:
[This](https://discourse.canonical.com/t/ssdlc-v1-1-is-here/4347) is the discourse post asking for it and this is an [internal](https://docs.google.com/document/d/1Gjlfe5M-X2ZgGFoKQ7uUhBpCrKhUCVuljK3xK2_RXiY/edit?tab=t.0#heading=h.esvwnqpeu35u) document (already approved) and [this](https://docs.google.com/document/d/1RolsiWCWhM-sVmHFjLt-Mq5yKcgQJ8GLXssGJREjSL0/edit?tab=t.0) is the spec

Reply:
Ah excellent! Could you add these links to the commit message and PR description so we can reference the source later?

Reply:
Done

Reply:
Thanks :)